modules/RTC/ScreenObtainer.js

```diff
@@ -212,13 +237,19 @@ const ScreenObtainer = {
             }
         }
 
+        // Allow a user to be shown a preference for what screen is to be captured.
+        if (browser.isSafari() && browser.isEngineVersionGreaterThan(11) && desktopDisplaySurface) {
```

Comment:
We don't support Safari < 13 so you can skip the version check.

Reply:
Thank you! Updated to remove that